app/views/contexts/section_summary_context.py

```diff
@@ -10,8 +10,8 @@
 
 
 class SectionSummaryContext(Context):
-    def __call__(self, current_location):
-        summary = self._build_summary(current_location)
+    def __call__(self, current_location, return_to="summary"):
```

Comment:
Thought on `return_to` being an attribute of `SectionSummaryContext` to remove some unneeded passing around of the arg and to avoid hardcoding to `summary` in multiple places?

Reply:
It considered it, but thought it might look a bit odd with the an __init__ with just return_to in, as I wouldn't add it to Context. Might do it and see how it looks

Reply:
Had a look, but no real benefit, SectionSummaryContext isn't passing into group, just some of its attributes,  which is called on _build_summary in SectionSummaryContext, so it would save one arg pass, but add an __init__